test/health_test.go

```diff
@@ -158,12 +169,17 @@ func TestHealthMetrics(t *testing.T) {
 	statusMetric := metrics[tally.KeyForPrefixedStringMap("inbound.calls.status.200", tags)]
 	value = *statusMetric.MetricValue.Count.I64Value
 	assert.Equal(t, int64(1), value, "expected counter to be 1")
+
+	loggedMetrics := metrics[tally.KeyForPrefixedStringMap("zap.logged.info", defaultTags)]
+	value = *loggedMetrics.MetricValue.Count.I64Value
+	assert.Equal(t, int64(2), value, "expected counter to be 2")
 }
 
 func TestRuntimeMetrics(t *testing.T) {
 	gateway, err := testGateway.CreateGateway(t, nil, &testGateway.Options{
 		CountMetrics:         true,
 		EnableRuntimeMetrics: true,
+		MaxMetrics:           31,
```

Comment:
maybe hoist https://github.com/uber/zanzibar/pull/231/files#diff-5a9b0fba1adefb12f372c4444b3ef8e6R194, and set the value to `numMetrics`?